app/grandchallenge/core/static/css/core.css

```diff
@@ -10,6 +10,10 @@ p {
     line-height: 1.6;
 }
 
+.child-mb-0>* {
```

Comment:
Seems a bit too non-specific? Or is this a problem in many places?

Reply:
I thought to use the bootstrap approach and describe the class with what it does. An alternative is to use a very specific 'rendered-result-text' class, but that seemed totally out of place here.

I for see we might want to do this in other places in the future.

Reply:
Doesn't MarkDown wrap everything in `<p>` so we only need to select that?

Reply:
It does. We can make it a bit more specific with `.child-p-md-0>p`.